### DISCUSSION TITLE
Question about how themes work

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#750] [Q&A] [answered]
I have a question on using multiple themes 
How would do something like themes on different colorschemes?
so for every scheme (Dark and Light) There would be something like OneDark, NordDark and MantineDark, so different itterations for different themes?

**Comment by rtivital** [comment, marked as answer]
You can try replacing colors, for example you can define `theme.colors.brand` and replace these colors when theme variant changes. You'll need to create state for theme variant on your own.
